res/css/views/settings/_JoinRuleSettings.pcss

```diff
@@ -53,6 +53,14 @@ Please see LICENSE files in the repository root for full details.
         display: block;
     }
 
+    &.mx_StyledRadioButton_disabled {
+        opacity: 0.5;
+    }
+
+    &.mx_StyledRadioButton_disabled + span {
+        opacity: 0.5;
+    }
```

Comment:
is this from a design? In general I'd prefer things to just be actual colours rather than rely on opacity blending with the background.

Reply:
It is not from design but a conscious decision. This way any styling inside the element (like the black title vs grey subtitle) will get adjusted accordingly.